seeding/automod.go

```diff
@@ -132,8 +134,85 @@ func initialize(ctx context.Context, logger *slog.Logger, ac *crcon.AutoModSeedi
 				},
 			},
 		},
+		// Allowlist deadzonefences: Horizontal A2 to J2
```

Comment:
This looks more like A2 _and_ J9, instead of _to_.

Meaning, you probably wanted to have a fence more like this:
```
		{
			Y: internal.Pointer(2),
			Condition: &data.Condition{
				Equals: map[string][]string{
					"map_name": append([]string{}, axisToAlliedHorizontalMaps..., alliedToAxisHorizontalMaps...),
				},
			},
		},
```

And then remove the second fence added (the X: J, Y: 2, one). Same for the vertical maps then as well 🤔